frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/CreateCluster.tsx

```diff
@@ -601,7 +612,14 @@ export default function CreateCluster(props: { infrastructureType: ClusterInfras
       break
     case Provider.kubevirt:
       template = Handlebars.compile(kubevirtTemplate)
-      controlData = getControlDataKubeVirt(t, handleModalToggle, <Warning />, isACMAvailable, localCluster)
+      controlData = getControlDataKubeVirt(
+        t,
+        handleModalToggle,
+        <Warning />,
+        isACMAvailable,
+        localCluster,
+        hypershiftSupportedVersions.slice(0, 3)
```

Comment:
Is this really needed? I think we should allow any version from the ConfigMap.

Reply:
I thought the issue said only the current version and 2 down

Reply:
If HyperShift supports a greater range than that, I'd prefer to allow it. We often have support exceptions for older versions. The main issue was that the N+1 will not be in this list.